engine/tests/phpunit/unit/Elgg/Notifications/NotificationsServiceUnitTestCase.php

```diff
@@ -102,13 +100,10 @@ public function down() {
 
 	public function setupServices() {
 		$this->notifications = new NotificationsService(
-			$this->subscriptions,
 			$this->queue,
 			$this->hooks,
 			$this->session,
-			$this->translator,
-			$this->entities,
-			$this->logger
+			$this->entities
```

Comment:
check the constructor vars please, I believe you missed some cleanup

Reply:
done